### ISSUE TITLE
When keyboard focus is on "Korean egg roll" recipe of my recipe list and press ENETR/SPACE key recipe is not opening: A11y_.NET Core_Recipes App My Recipes_My Recipes_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#22823]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags
#A11yTCS;#A11yMAS;#WCAG2.1.1;#Keyboard;#E2E_.NETCore_Win32_Jun2024;#.NET Core;#DesktopApp;#Win32;#FTP;#A11ySev2;AILimted; #Closed; #Regressed:10-03-24;

## Environment Details:
Application: .NET Core
Version .NET 9
Operating System: Window 11 version 23H2 OS build 22631.2861

## Repro Steps:

1. Install and open ".NET MAUI" app.
2. TAB to "My Recipes" and Press ENTER key.
3. TAB to "Korean egg roll" recipe and press ENTER/SPACE key to open.
5. Observe that when keyboard focus is on "Korean egg roll" recipe of my recipe list and press ENETR/SPACE key recipe is not opening

## Actual Result:
When keyboard focus is on "Korean egg roll" recipe of my recipe list and press ENETR/SPACE key recipe is not opening.
## Expected Result:
When keyboard focus is on "Korean egg roll" recipe of my recipe list and press ENETR/SPACE key recipe should be open.
## User Impact:
Keyboard only user will not be able open recipe. 
### Attachment
![22823_11y_ NET Core_Recipes App My Recipes_My Recipes_Keyboard](https://github.com/dotnet/maui/assets/124254981/03674c17-3117-431e-b970-1791c5f3dace)

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio 17.11.0 Preview 1.1(8.0.40 & 8.0.21 &8.0.3 & 8.0.60-ci.net8.24304.1). Can repro on Windows platform with Recipes sample project.

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by kapilvaishna** [comment]
Issue is not repro in version 3(3) so closing the bug.

https://github.com/user-attachments/assets/d234f672-405e-4aa6-967a-bdb095d465d8
